examples/browser-vue/babel.config.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
   presets: ['@vue/app'],
+  plugins: ['@babel/plugin-transform-runtime']
```

Comment:
While this fixed the issue in the remaining examples, I could not make this work in Vue. It seems that this plugin is not correctly getting to the babel config